### ISSUE TITLE
evaluation

### ISSUE THREAD

**Issue by shainaraza** [UKPLab/sentence-transformers#169]
Thanks for sharing the great work and library, I have a confusion in my understanding of evaluation , please clarify for my understanding.
The evaluation metrics can only be used during pre-training? if I fine-tune a model for some task, lets say k-mean clustering, so how would I evaluate my results.
thanks in advance

**Comment by nreimers** [comment]
Hi @shainaraza 
You would need to implement the evaluation yourself for your specific task.

For clustering for example, there is no standard way of evaluation. So you would need to find a way that fits your data & task.

Sure you can tune on any data set. But often only certain data sets & loss functions are useful. Fine-tuning is  sadly still quite complex, as we target unsupervised tasks where we don't have labels and a loss we could propagate. So it is always trying around different things.

Best
Nils Reimers

**Comment by shainaraza** [comment]
thanks you very much
